Rakefile

```diff
@@ -0,0 +1,6 @@
+require "bundler/gem_tasks"
```

Comment:
I'm ashamed to say this is voodoo programming to me. I guess I need to read up on Rakefiles.